starter_code/src/movies.js

```diff
@@ -2,18 +2,77 @@
 
 // Iteration 1: All rates average - Get the average of all rates with 2 decimals 
 
+function ratesAverage(arr){ 
+let sum = arr.reduce(function(acc, value){
+    return acc + parseFloat(value.rate);
+}, 0); return sum / arr.length;
+};
  
 // Iteration 2: Drama movies - Get the average of Drama Movies
 
+function dramaMoviesRate(arr) {
+    let dramaMovies = arr.filter(function(movie){
+        if(movie.genre.indexOf("Drama") !== -1) {
+            return true;
+        }
+    } ); if(dramaMovies.length === 0) {
+        return 0};
+    let sum = dramaMovies.reduce(function(acc, value){
```

Comment:
Here you could use the function you wrote in iteration 1 and just input the dramaMovies as your argument